README.md

```diff
@@ -3,6 +3,14 @@ GOV.UK Frontend ·
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 =====================
 
+> **Warning**
+>
+> We’re currently implementing heavy changes to our code for [our next major version](https://github.com/alphagov/govuk-frontend/milestone/46).
+>
+> We recommend you wait before forking this repository to limit your effort of keeping your fork up to date with our changes.
```

Comment:
One for @claireashworth but should we combine the two paragraphs into one?

```suggestion
> We recommend waiting before forking this repository to avoid frequent updates. If you're fixing a bug in our previous releases, branch off the [latest tag](https://github.com/alphagov/govuk-frontend/tags) instead.
```

Reply:
Yeah nearly, perhaps the abrupt "Do not fork" loses the sense of temporarily waiting until we've finished? It's not likely, but don't want it sounding like people can never fork again

Reply:
I see. We've already talked about the changes going on so how about:

While we're making changes, only fork this repository if necessary. This is to limit how often you will need to update your fork to keep up to date.

Reply:
I like that, but now I realise it's longer than Romaric's original and I only commented to keep it short and sweet 😆

Sorry both